src/nunavut/lang/c/templates/UnionType.j2

```diff
@@ -10,11 +10,13 @@
  * a(n) {{ field.name }} type. If so then {{ composite_type | full_reference_name }}->{{ field.name }}
```

Comment:
```suggestion
 * {{ field.name }}. If so, then {{ composite_type | full_reference_name }}->{{ field.name }}
```

Changed the wording because it's not the type we're checking but the variant. There may be multiple variants sharing the same type.